### PR TITLE
Debugger: show CIA in register view

### PR DESCRIPTION
**Pull request by illusion0001**
Copying CIA instead of typing it in is easier for later use

### CODE REVIEW

File: rpcs3/Emu/Cell/PPUThread.cpp [hunk 1]
```diff
@@ -1192,7 +1192,7 @@ void ppu_thread::dump_regs(std::string& ret) const
 		}
 	}
 
-	fmt::append(ret, "CIA: 0x%08llx\n", cia);
+	fmt::append(ret, "CIA: 0x%08x\n", cia);
```
Comment (elad335) [inline rpcs3/Emu/Cell/PPUThread.cpp:1195]: 0x%07x suffices. It's rarely above 256mb.

Comment (Nekotekina) [reply rpcs3/Emu/Cell/PPUThread.cpp:1195]: Won't it be ugly this way?

Comment (elad335) [reply rpcs3/Emu/Cell/PPUThread.cpp:1195]: The always added 0 in logs looked odd there IMO. It's also how the debugger prints PPU instruction addresses now in which the added 0 looked odd there as well.

Comment (illusion0001) [reply rpcs3/Emu/Cell/PPUThread.cpp]: Removed 0s prefix

**Review by Scalerize — APPROVED** [review]

**Pull request merged**